src/app_engine/constants.py

```diff
@@ -42,6 +42,21 @@ class EventType(object):
   # Event signifying that a room enters the state of having exactly
   # two participants.
   ROOM_SIZE_2 = 'room_size_2'
+  ICE_CONNECTION_STATE_CONNECTED = 'ice_connection_state_connected'
+
+
+class AnalyticsField(object):
```

Comment:
Are you suggesting something like `ANALYTICS_FIELD_TYPE` or something?

I am happy to move it to analytics. In that case we way want to move LogField too; I know you suggested this before but we never resolved the issue.